python_purify/core.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, api_key, live=True, rspformat='json', verbose=False,
     @staticmethod
     def _make_options(**kwargs):
         out = []
-        for key, value in kwargs.iteritems():
+        for key, value in kwargs.items():
```

Comment:
https://stackoverflow.com/questions/27580938/python-2-and-3-compatible-way-of-iterating-through-dict-with-key-and-value